react/package.json

```diff
@@ -8,20 +8,21 @@
     "@types/node": "^12.12.31",
     "@types/react": "^16.9.26",
     "vtex.address-context": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.address-context@0.0.4/public/@types/vtex.address-context",
-    "vtex.checkout-components": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-components@0.1.0/public/@types/vtex.checkout-components",
+    "vtex.checkout-components": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-components@0.3.0/public/@types/vtex.checkout-components",
     "vtex.checkout-container": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-container@0.4.0/public/@types/vtex.checkout-container",
-    "vtex.checkout-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-graphql@0.32.1/public/@types/vtex.checkout-graphql",
+    "vtex.checkout-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-graphql@0.34.2/public/@types/vtex.checkout-graphql",
     "vtex.format-currency": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.format-currency@0.1.3/public/@types/vtex.format-currency",
     "vtex.formatted-price": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.formatted-price@0.4.0/public/@types/vtex.formatted-price",
     "vtex.order-manager": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.order-manager@0.8.4/public/@types/vtex.order-manager",
     "vtex.order-shipping": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.order-shipping@0.4.1/public/@types/vtex.order-shipping",
-    "vtex.place-components": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.place-components@0.7.0/public/@types/vtex.place-components",
+    "vtex.place-components": "https://shp--checkoutio.myvtex.com/_v/private/typings/linked/v1/vtex.place-components@0.7.0+build1590695461/public/@types/vtex.place-components",
```

Comment:
You need to revert these back
```
error:
Error on react package.json: Inconsistent app context for 'vtex.place-components' types:
Expected 'master' for 'workspace' but received 'shp'
Expected 'vtex' for 'account' but received 'checkoutio'
Running 'vtex setup' may solve this.
```

Reply:
actually I'll need to wait for vtex-apps/place-components#14 before merging this, because I'm using some new props that exist on that PR only :-/